Stratis.Bitcoin.Features.Consensus/PosConsensusValidator.cs

```diff
@@ -290,58 +278,44 @@ public override void ContextualCheckBlockHeader(ContextInformation context)
             }
 
             // Check coinbase timestamp.
-            if (chainedBlock.Header.Time > FutureDrift(context.BlockResult.Block.Transactions[0].Time, chainedBlock.Height))
+            if (chainedBlock.Header.Time > this.FutureDrift(context.BlockResult.Block.Transactions[0].Time))
             {
                 this.logger.LogTrace("(-)[TIME_TOO_NEW]");
                 ConsensusErrors.TimeTooNew.Throw();
             }
 
             // Check coinstake timestamp.
             if (context.Stake.BlockStake.IsProofOfStake()
-                && !PosConsensusValidator.CheckCoinStakeTimestamp(chainedBlock.Height, chainedBlock.Header.Time, context.BlockResult.Block.Transactions[1].Time))
+                && !this.CheckCoinStakeTimestamp(chainedBlock.Height, chainedBlock.Header.Time, context.BlockResult.Block.Transactions[1].Time))
             {
                 this.logger.LogTrace("(-)[BAD_TIME]");
                 ConsensusErrors.StakeTimeViolation.Throw();
             }
 
             // Check timestamp against prev.
-            if ((chainedBlock.Header.Time <= StakeValidator.GetPastTimeLimit(chainedBlock.Previous))
```

Comment:
What happened to this check?
```
FutureDrift(chainedBlock.Header.Time, chainedBlock.Height) < chainedBlock.Previous.Header.Time))		
```

Reply:
It should be described in the commit that removed that. That check is void, refer to that commit description pls.

Reply:
ah ok I need to find it its not visible when viewing the entire set

Reply:
Ah ok yeah once you strip away the GetPastTimeLimit its more clear the second condition has no effect.